GOVERNANCE.md

```diff
@@ -0,0 +1,67 @@
+# Bank-Vaults Project Governance
+
+Bank-Vaults is an open source project dedicated to simplifying the complex world of secret management in the Cloud Native ecosystem and on Kubernetes. This document outlines the project's governance structure and how community members can contribute and participate in decision-making processes.
+
+## Community Roles
+
+### Users
+
+Users are the foundation of the Bank-Vaults community. They utilize the project in their environments and provide valuable feedback. We encourage users to:
+
+- Share their experiences with Bank-Vaults
+- Report issues and suggest improvements
+- Participate in discussions on our communication channels
+- Contribute to documentation by reporting unclear or outdated sections
+
```

Comment:
Maybe we should add a line here about users of Bank-Vaults are encouraged to add themselves to our [Adopters list](https://github.com/bank-vaults/bank-vaults/blob/main/ADOPTERS.md)?